lib/daru/dataframe.rb

```diff
@@ -2004,6 +2004,23 @@ def split_by_category cat_name
       end
     end
 
+    # returns array of row tuples at given index(s)
+    def access_row_tuples_by_indexs *indexes
```

Comment:
This method returns rows . This is inspired from the private method [`access_row`](https://github.com/SciRuby/daru/pull/330/files#diff-cfd587da29af6a7a2d1f98c614ef123fL2106), which returns column in `Vector`.